taxonomy/views.py

```diff
@@ -0,0 +1,79 @@
+"""
+Views for the taxonomy app
+"""
+
+from django.shortcuts import render, redirect
+
+from .forms import VocabularyForm
+from learningresources.models import Repository
+from taxonomy.models import Vocabulary
+
+
+def edit(request, vocabulary_id=None):
```

Comment:
Also, is 'edit' an appropriate name for a view here? What is the Django preferred way?